app/policies/idv/gpo_verify_by_mail_policy.rb

```diff
@@ -1,24 +1,34 @@
 # frozen_string_literal: true
 
 module Idv
-  class GpoMail
-    attr_reader :current_user
+  class GpoVerifyByMailPolicy
+    attr_reader :user
```

Comment:
This is fun: initially this was a new class, but then I moved over enough from GpoMail that GitHub has solved the Ship of Theseus problem for us.